storage/ndb/include/ndbapi/NdbAggregationCommon.hpp

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright [2024] <Copyright Hopsworks AB>
```

Comment:
Change this line in all files u changed to:
   Copyright (c) 2024, 2024, Hopsworks and/or its affiliates.